HW04/anna_tsekhanskaya/lib/mentor.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Mentor
+  atter_accessor :name, :surname, :noties_for_student, :hw
+  def initialize
+    @name = name
+    @surname = surname
+    @noties_for_student = []
+    @list_students = []
+    @hw = []
+  end
+
+  def full_name
+    "#{name.capitalize} #{surname.capitalize}"
+  end
+
+  # mentor add new homework
+  def add_homework(title, description, student)
+    noties_for_student << Notification.new(title: title,
+                                           description: "#{mentor.full_name} added hw at #{Time.now}", student: student)
+    hw << Homework.new(title: title, description: description, student: Student.full_name, mentor: Mentor.full_name)
+  end
+
+  # mentor subscribe to student
+  def subscribe_to
+    list_students << Student.full_name
+  end
+
+  # mentor see notification about homework to work
+  # mentor see notification about homework to check
+  def notifications
```

Comment:
There's no need in this method.

Reply:
Once again :) There's no need in this method at all. To make `mentor.notifications` and `student.notifications` work you need to change the name of `noties_for_student` & `noties_for_mentor`  and a little of its usage.